package.json

```diff
@@ -37,6 +37,7 @@
     "storybook": "storybook dev -p 6006",
     "test": "karma start spec/karma.conf.js",
     "test:jest": "jest",
+    "test:smoke": "playwright test",
```

Comment:
At some point it would be include something about this in the Readme, how to set up/run these tests locally (if we intend to make them runnable locally) or how it's wired in the CI.

Reply:
At this stage, I don’t find it important to make this runnable locally; It doesn’t make sense in its current state. Of course, I can document how to run it in a CI environment if necessary.